presto-benchmark-runner/src/test/java/com/facebook/presto/benchmark/executor/TestConcurrentPhaseExecutor.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.benchmark.executor;
+
+import com.facebook.presto.benchmark.framework.BenchmarkQuery;
+import com.facebook.presto.benchmark.framework.BenchmarkQueryExecutor;
+import com.facebook.presto.benchmark.framework.BenchmarkRunnerConfig;
+import com.facebook.presto.benchmark.framework.QueryException;
+import com.facebook.presto.benchmark.framework.QueryResult;
+import com.facebook.presto.benchmark.prestoaction.JdbcPrestoAction;
+import com.facebook.presto.benchmark.prestoaction.PrestoAction;
+import com.facebook.presto.benchmark.prestoaction.PrestoClusterConfig;
+import com.facebook.presto.benchmark.prestoaction.PrestoExceptionClassifier;
+import com.facebook.presto.benchmark.retry.RetryConfig;
+import com.facebook.presto.jdbc.QueryStats;
+import com.facebook.presto.spi.ErrorCodeSupplier;
+import com.facebook.presto.sql.parser.SqlParser;
+import com.facebook.presto.sql.parser.SqlParserOptions;
+import com.facebook.presto.sql.tree.Statement;
+import com.facebook.presto.tests.StandaloneQueryRunner;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static com.facebook.presto.benchmark.BenchmarkTestUtil.CATALOG;
+import static com.facebook.presto.benchmark.BenchmarkTestUtil.SCHEMA;
+import static com.facebook.presto.benchmark.BenchmarkTestUtil.setupPresto;
+import static com.facebook.presto.spi.StandardErrorCode.GENERIC_INTERNAL_ERROR;
+import static com.facebook.presto.sql.parser.IdentifierSymbol.AT_SIGN;
+import static com.facebook.presto.sql.parser.IdentifierSymbol.COLON;
+import static java.util.Objects.requireNonNull;
+
+public class TestConcurrentPhaseExecutor
+{
+    private static SqlParser sqlParser;
+    private static StandaloneQueryRunner queryRunner;
+    private static Map<String, BenchmarkQuery> queryMap;
```

Comment:
This should be private static final. Also, do not name variable with its type, call it `ALL_QUERIES`